docs/general/ad-filtering/create-own-filters.md

```diff
@@ -628,13 +622,12 @@ The rule applies only to WebRTC connections.
 * [`$content`](#content-modifier)
 * [`$elemhide`](#elemhide-modifier)
 * [`$extension`](#extension-modifier)
+* [`$genericblock`](#genericblock-modifier)
+* [`$generichide`](#generichide-modifier)
 * [`$jsinject`](#jsinject-modifier)
 * [`$specifichide`](#specifichide-modifier)
 * [`$stealth`](#stealth-modifier)
 * [`$urlblock`](#urlblock-modifier)
-* [Generic rules](#exception-modifiers-generic-rules)
-  * [`$genericblock`](#genericblock-modifier)
-  * [`$generichide`](#generichide-modifier)
```

Comment:
list items are moved up but their paragraphs are not moved

Reply:
@slavaleleka I don't understand what's wrong here?

Reply:
@sxgunchenko asked to put all the modifiers in a sorted order

Reply:
so now the list is sorted but the anchor to `#exception-modifiers-generic-rules` section (common info for $genericblock and $generichide) is missed

Reply:
@slavaleleka now i see that i changed it in the separated section, got you, thanks

Reply:
fixed